.solcover.js

```diff
@@ -1,6 +1,13 @@
 module.exports = {
+  file: 'test/*.js',
   norpc: true,
   copyPackages: ['openzeppelin-solidity'],
   skipFiles: ['Migrations.sol'],
-  providerOptions: { total_accounts: 30, seed: 0, network_id: 420 },
+  istanbulReporter: ['html', 'lcov', 'text'],
+  providerOptions: {
+    mnemonic: 'square include clarify skin garden tube tide eight eternal grit hybrid library',
```

Comment:
Again, do not include mnemonic here. For solcover, I don't think we even need to provide a mnemonic.
See the docs for the hardhat-solcover plugin, it should automatically use the hardhats configured provider.